--- aom-1.0.0/CMakeLists.txt~	2018-08-03 20:40:18.000000000 +0200
+++ aom-1.0.0/CMakeLists.txt	2018-08-03 20:49:19.111062025 +0200
@@ -259,7 +259,7 @@ add_library(aom_encoder_stats OBJECT ${A
 add_library(aom ${AOM_SOURCES} $<TARGET_OBJECTS:aom_rtcd>)
 
 if(NOT MSVC AND NOT APPLE)
-  target_link_libraries(aom ${AOM_LIB_LINK_TYPE} m)
+  target_link_libraries(aom ${AOM_LIB_LINK_TYPE} m pthread)
 endif()
 
 # List of object and static library targets.
